pkgs/tools/networking/maubot/default.nix

```diff
@@ -60,12 +41,12 @@ let
 
   maubot = python.pkgs.buildPythonPackage rec {
     pname = "maubot";
-    version = "0.4.1";
-    disabled = python.pythonOlder "3.8";
+    version = "0.4.2";
+    disabled = python.pythonOlder "3.9";
```

Comment:
Why? The readme still says 3.8 minimum: https://docs.mau.fi/maubot/usage/setup/index.html

Reply:
https://github.com/maubot/maubot/blob/75879cfb9370aade6fa0e84e1dde47222625139a/setup.py#L44